common/buildings/wc_common_buildings.txt

```diff
@@ -4,16 +4,19 @@ aviary_complex_01 = {
 	construction_time = quick_construction_time
 
 	can_construct_potential = {
+		building_aviary_complex_requirement_terrain = yes
 		building_requirement_castle_city_church = { LEVEL = 01 }
-		OR = {
-			scope:holder.culture = {
-				OR = {
-					has_innovation = innovation_taming_the_skies
-					has_innovation = innovation_flying_beasts
-				}
+		culture = {
+			OR = {
+				scope:holder.culture = {
+					OR = {
+						has_innovation = innovation_taming_the_skies
+						has_innovation = innovation_flying_beasts
+					}
+				}	
+				has_special_aerial_maa_trigger = yes
```

Comment:
I see it checks for culture now. Not sure it's good from gameplay perspective. It allows you to build a building you can't use because you don't have MaA.

Reply:
All of the triggers check that the MaA is unlocked, either the two basic ones or they have a special one. Is there something I'm missing?

Reply:
> All of the triggers check that the MaA is unlocked, either the two basic ones or they have a special one. Is there something I'm missing?

No, this trigger checks for culture, not MaA.

Reply:
has_special_xxxx_trigger only includes those who already have the maa at the start. 583. It will only cause an issue if we go back on the dates